boards/google/quincy/google_quincy.dts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022 Google LLC
```

Comment:
Wrong year in the copyright here and in several other files.

Reply:
Updated, thanks